client/my-sites/plugins/main.jsx

```diff
@@ -84,10 +85,13 @@ const PluginsMain = React.createClass( {
 	},
 
 	getPluginsState( nextProps ) {
+		const props = nextProps || this.props;
```

Comment:
It might be more conventional — and less confusing? — to change the signature of the method to `getPluginsState( props )`, not have the method default to `this.props` and instead place that responsibility on whatever is calling the method — `this.getPluginsState( nextProps )`